api/src/OpenTelemetry/Logging/Core.hs

```diff
@@ -22,24 +22,29 @@ module OpenTelemetry.Logging.Core (
   shortName,
   severityInt,
   emitLogRecord,
+  addAttribute,
+  addAttributes,
+  logRecordGetAttributes,
 ) where
 
 import Control.Applicative
 import Control.Monad.Trans
 import Control.Monad.Trans.Maybe
 import Data.Coerce
+import Data.HashMap.Strict (HashMap)
 import qualified Data.HashMap.Strict as H
 import Data.IORef
 import Data.Maybe
+import Data.Text (Text)
 import GHC.IO (unsafePerformIO)
-import OpenTelemetry.Attributes (AttributeLimits)
 import OpenTelemetry.Common
 import OpenTelemetry.Context
 import OpenTelemetry.Context.ThreadLocal
 import OpenTelemetry.Internal.Common.Types
 import OpenTelemetry.Internal.Logging.Types
 import OpenTelemetry.Internal.Trace.Types
-import OpenTelemetry.LogAttributes
+import OpenTelemetry.LogAttributes (ToValue)
+import qualified OpenTelemetry.LogAttributes as A
```

Comment:
Changed to a qualified import so that `addAttributes` does not cause a naming conflict. It would be possible to rename `addAttributes` to `logRecordAddAttributes`, but `OpenTelemetry.Trace.Core. addAttributes` is not named `spanAddAttributes`.